include/glow/Graph/Graph.h

```diff
@@ -849,6 +849,13 @@ class Function final : public Named {
   BatchOneHotNode *createBatchOneHot(llvm::StringRef name, NodeValue data,
                                      NodeValue lengths, NodeValue values);
 
+  /// Given Input tensor of [N,H,W,C], where N is the batch
+  /// axis, H is the height, W is
+  /// the width, C is the channel or depth. This produces Output tensor of [N,
+  /// H/blockSize, W/blockSize, C * blockSize * blockSize].
+  SpaceToDepthNode *createSpaceToDepth(llvm::StringRef name, NodeValue input,
+                                       unsigned blocksize);
```

Comment:
```suggestion
                                       unsigned blockSize);
```
All the other methods in this file follow camelCaseStyle for argument names. Let's follow it in this method as well, please.